src/libraries/System.Net.Security/src/System/Net/Security/SslStream.cs

```diff
@@ -863,6 +863,17 @@ private void ThrowIfExceptionalOrNotAuthenticated()
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Are the call sites very hot paths?  Wondering if it's really beneficial to force inline it like this.

Reply:
I can try to measure it. For now, I just followed pattern from other helper methods. 
Generally I would not expect them on hot path. The properties would either never be used or used once for logging after handshake is completed.